docs/reference/identities.md

```diff
@@ -0,0 +1,37 @@
+# Identities
+
+Pebble has the concept of named "identities", which lets administrators allow additional users to access the [API](../explanation/api-and-clients.md).
```

Comment:
Hi @benhoyt , @IronCore864 ,

From the current implementation it seems to make sense to use identities to give specific users elevated access to the APIs. But "allow additional users" doesn't seem so accurate since you can (theoretically) force an identity's access to "untrusted" right?

So maybe reword "allow additional users to access the API" to "manage users' access to the API"?

Thanks.

Reply:
Good suggestion -- thanks.